django_ipam/api/urls.py

```diff
@@ -3,7 +3,10 @@
 from . import views
 
 urlpatterns = [
-    url(r'^subnet/(?P<subnet_id>[^/]+)/request-ip/$',
-        views.request_ip,
+    url(r'^subnet/(?P<subnet_id>[^/]+)/get-first-available-ip/$',
+        views.get_first_available_ip,
+        name='get_first_available_ip'),
+    url(r'^subnet/request-ip/$',
```

Comment:
it should be either `subnet/<subnet_id>/request-ip` or just `request-ip/`, this way it's not very restful

Reply:
I suggest to keep consistency with the previous one and have `subnet/<subnet_id>/request-ip` otherwise it looks weird, it shouldn't be hard with DRF, you'll find the param in the function arguments